### ISSUE TITLE
Lite version for existing Pixel users?

### ISSUE THREAD

**Issue by Kylejustknows** [Kingsman44/Pixelify#171]
As the existing Pixel6/7 users, we already have most of the "Pixelify" functions from the factory. The only function we probably need is the "unlimited upload".

(When installing the current Pixelify over pixel devices, even picking everything "No", things would get messed up..... E.g. picking no to "call screen", the previously working "call screen" would disappear. Picking "yes" then many things would rollback to older versions. It is getting frustrating.)

I would love to see a "lite version" for existing pixel 6/7 users. 

Or, could the developer give me a simple hint that which files are involved?
 (I tried to stripe the pixelify myself but found the zip contains a massive number of files for all functions. Could not rule out many of them)

**Comment by pgamerx** [comment]
> Totally understand this frustration, Would really love to see a lite version for the same,
> 
> because with the current module, the assistant typing, now playing, quick phrases get messed up really badly!!!

I agree. It also messed up with NGA stuff and the sub modules are unreliable. I tried using the dialer mod to get call screening to work in unsupported regions but couldn't. Hopefully they see this
